### DISCUSSION TITLE
Managing noisy pots

### DISCUSSION THREAD

**Discussion by archaro** [GrandOrgue/grandorgue#1121] [Developers] [answered]

**Comment by midimusic** [comment]
First of all what EXACTLY is it that you are calling a "pot"? And what are they all used for? The most common use is for swell and crescendo controls.

The normal electronic meaning of "pot" is an abbreviation of "potentiometer", these are usually rotary variable resistors. Is that what you mean? and if so why do you have five of them? However I will assume for now that is what you do mean.

It is very unlikely that your problem is due to interference picked up in the wiring much more likely that the "pots" are old and worn out. Do you get the midi messages only when you move/adjust a pot setting? If so which one and what is it used for?

Midi sources are normally debounced at source before they get a chance to make midi messages so your situation is very unusual. The fact that your midi messages are always the the same indicates that they are not being produced by noise but intentionally by some source which you are unaware of - possibly a fault code.

Some midi keyboards produce a continuous flow of "Active Sensing" midi messages just to let the system know that they are still connected. Such messages are routinely ignored and cause no problems.

It is very unlikely that GO will be changed just to cope with your particular problem - you need to fix it at your end!! However GO will already be ignoring these signals unless you have set it up to use them.

csw900

**Comment by archaro** [comment, marked as answer]
I had written a very long response, explaining myself more clearly and defending my feature request.  Then I realised that it would be a waste of time.  I apologise for the inconvenience, and will bother you no more.
